mix.exs

```diff
@@ -9,7 +9,8 @@ defmodule Plug.Mixfile do
      package: package,
      description: "A specification and conveniences for composable " <>
                   "modules in between web applications",
-     docs: [readme: true, main: "README"]]
+     docs: [readme: true, main: "README",
+            source_url: "https://github.com/elixir-lang/plug"]]
```

Comment:
Can you add `source_ref` as in here? https://github.com/elixir-lang/ecto/blob/master/mix.exs#L25-L26